README.rst

```diff
@@ -542,10 +542,10 @@ bodies.
                                        'UTF-8.')
 
         def process_response(self, req, resp, resource):
-            if 'result' not in resp.context:
```

Comment:
🤔 It may make sense to do a followup PR to change the attribute name to ``doc`` to improve the symmetry between the request and response examples.

Reply:
Right, I will revise these in a followup PR:
* ``README.rst``, ``quickstart.rst`` and ``test_example.py`` all contain slightly different versions of the same file
* The example is not tested (bar ``flake8`` on ``test_example.py``)

Reply:
I have created a followup issue: https://github.com/falconry/falcon/issues/1423